ProcessMaker/processmaker
# FOUR-20916: Create a Dashboard for Monitoring

## Issue & Reproduction Steps
Describe the issue this ticket solves and describe how to reproduce the issue (please attach any fixtures used to reproduce the issue).

## Solution
- Create PrometheusMetricsManager class to manage the metrics
- migrate the logic for metrics
- add prometheus and grafana

## How to Test
- run 
`composer install`
`docker compose up -d`
see next available resources
`http://processmaker.test/metrics`

prometheus
`http://localhost:9090/targets`
grafana

`http://localhost:3000/`


## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-20916

## Code Review Checklist
